tests/EStimLibrary.UnitTests/Extensions/SpatialModel/StringHierarchy/StringHierarchyRegionTests.cs

```diff
@@ -0,0 +1,547 @@
+﻿using EStimLibrary.Extensions.SpatialModel.StringHierarchy;
+
+
+namespace EStimLibrary.UnitTests.Extensions.SpatialModel.StringHierarchy;
+
+
+/// <summary>
+/// Test class for StringHierarchyRegion.
+/// </summary>
+public class StringHierarchyRegionTests
+{
+    private readonly ITestOutputHelper _output;
+
+    // Test class constructor creates an output helper so can write console
+    // output.
+    public StringHierarchyRegionTests(ITestOutputHelper testOutputHelper)
+    {
+        this._output = testOutputHelper;
+    }
+
+    // Test method naming convention: LibClassMethodName_ScenarioShouldExpect
+
+    /// <summary>
+    /// Test the constructor with null parameter values.
+    /// </summary>
+    [Fact]
+    public void Constructor_ShouldAcceptNull()
+    {
+        var stringHierarchyRegion = new StringHierarchyRegion("base", null,
```

Comment:
Nice test cases from a first quick look! Would you mind just formatting them - i.e., all lines <=80 characters, kinda like I've done in the first few sections here - and adding some in-line comments throughout your method bodies to help me quickly get a grasp of what you're testing in each? Both what structures you're making and then what you're asserting?